### ISSUE TITLE
Install Build Tools Problem

### ISSUE THREAD

**Issue by wraseman** [rstudio/packrat#500]
Hi there, 

I am having issues selecting the "Use packrat with this project" option in the "Project Options" dialog. Every time, the following "Install Build Tools" dialog pops up, I say yes, and it installs the new build tools. 

![image](https://user-images.githubusercontent.com/14048152/45765377-2c07f980-bbf2-11e8-9d55-3d3e72114d99.png)

When it has finished installing, the "Use packrat with this project" option is still unchecked, so I check it again just to get the same "Install Build Tools" dialog. I've tried this several times and have restarted the computer as well with no luck. 

Any help on this issue would be appreciated!

Thanks,
Billy

**Comment by imartinezl** [comment]
Yep, I have the same issue on my laptop. (Windows 10 version, but with R 3.3.3)
